apps/managedidentity/managedidentity.go

```diff
@@ -378,7 +424,6 @@ func (c Client) retry(maxRetries int, req *http.Request) (*http.Response, error)
 	var err error
 	for attempt := 0; attempt < maxRetries; attempt++ {
 		tryCtx, tryCancel := context.WithTimeout(req.Context(), time.Second*15)
```

Comment:
Why 15s? I recommend 60s to use the same default as .NET

Reply:
Azure SDK has it at 15 seconds, I will update it to 60